contrib/tesseract/template.py

```diff
@@ -0,0 +1,78 @@
+pkgname = "tesseract"
+pkgver = "5.4.1"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = [
+    "asciidoc",
+    "automake",
+    "pkgconf",
+    "slibtool",
+    "xsltproc",
+]
+makedepends = [
+    "icu-devel",
+    "leptonica-devel",
+    "libarchive-devel",
+    "libcurl-devel",
+    "pango-devel",
+]
+depends = [
+    "tesseract-data-eng",
+    "tesseract-data-osd",
+]
+pkgdesc = "OCR engine"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "Apache-2.0"
+url = "https://tesseract-ocr.github.io"
+source = f"https://github.com/tesseract-ocr/tesseract/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "c4bc2a81c12a472f445b7c2fb4705a08bd643ef467f51ec84f0e148bd368051b"
+# check: tests require external data
+options = ["!check"]
+
+
+def post_build(self):
+    # also build training tools
+    self.make.build(["training"])
+
+
+def post_install(self):
+    # also install training tools
+    self.make.install(["training-install"])
+
```

Comment:
you can technically put this in make_*_args and they'll be appended and work the same